werkomgeving/gegeven/index.md

```diff
@@ -5,8 +5,8 @@
 Een gegeven is *een vastgelegde uitspraak over een getypeerde eigenschap van een object*. Kenmerkend aan een gegeven zijn een aantal elementen:
 
 1. Een gegeven is **vastgelegd**: op papier, in steen, in een digitale vorm. In elk van deze vormen van sprake zijn van meer of minder mate van vluchtigheid of persistentie: gegevens in een bericht zullen minder persistent zijn dan gegevens die duurzaam worden opgeslagen in een administratie.
-2. Een gegeven is een **uitspraak**: zo'n uitspraak kan een bewering, stelling of waarneming zijn en zo'n uitspraak kan gedaan worden door een mens of een sensor. Ook kan zo'n uitspraak betrekking hebben op een berekening of afleiding.
-3. Een gegeven gaat over een **object**: het gaat over *iets*: de objecten die relevant zijn in het domein waarover we gegevens verwerken.
+2. Een gegeven gaat over een **object**: het gaat over *iets*: de objecten die relevant zijn in het domein waarover we gegevens verwerken.
+3. Een gegeven is een **uitspraak**: zo'n uitspraak kan een bewering, stelling of waarneming zijn en zo'n uitspraak kan gedaan worden door een mens of een sensor. Ook kan zo'n uitspraak betrekking hebben op een berekening of afleiding.
 4. Een gegeven betreft een **eigenschap** van dat object: het gegeven legt een uitspraak vast over een eigenschap die het object heeft. Dit kan zowel een kenmerk van het object zelf betreffen, als een relatie die een object heeft met een ander object.
```

Comment:
In verband hiermee is nog van belang dat een eigenschap (of juist een gegeven) benoemd is. Het heeft dus een ‘naam’. In onderstaand lijstje met voorbeelden zie je dat ook terugkomen. In de paragraaf over gegevensobject komt dat echter niet terug wat n.m.m. wel zou moeten.

Reply:
Ik snap niet helemaal wat je bedoelt met ‘naam’. Ik zie het ook niet in de voorbeelden. Een gegeven heeft m.i. geen naam. Kun je specifiek aangeven waar jij de naam in ziet, en waarom die van belang is?

Reply:
Wat zie jij dan precies als het gegeven? De totale zin zoals:

[Jan]’s leeftijd is 25

of toch alleen ‘25’?
In geval van dat laatste begrijp ik denk ik wat je bedoelt. Ik doelde op ‘leeftijd’ als naam maar dat is dan misschien de naam van de ‘eigenschap’.

Reply:
Inderdaad de naam van de eigenschap